cli/pkg/kctrl/cmd/app/release/imgpkg_runner.go

```diff
@@ -21,6 +21,7 @@ type ImgpkgRunner struct {
 	UseKbldImagesLock bool
 	ImgLockFilepath   string
 	UI                cmdcore.AuthoringUI
+	Tag               string
```

Comment:
Do we need this value here? Does not seem like it is getting used

Reply:
Doen